app/utils/goRoom.js

```diff
@@ -9,9 +9,9 @@ const navigate = ({ item, isMasterDetail, ...props }) => {
 	}
 
 	navigationMethod('RoomView', {
-		rid: item.rid,
+		rid: item.roomId || item.rid,
 		name: RocketChat.getRoomTitle(item),
-		t: item.t,
+		t: item.type ? 'p' : item.t,
```

Comment:
Why?

Reply:
API response for `teams.create` does not bring a type, it returns a boolean value (e.g. `type : true`).